resources/lib/constants.py

```diff
@@ -158,10 +163,35 @@
         "name": "Odia",
         "tvImg": IMG_PUBLIC + "logos/langGen/Odia_1579245819981.jpg",
         "promoImg": IMG_PUBLIC+"67/0/Odia_1580459753008_promo.jpg",
+    },
+    {
+        "name": "Assamese",
+        "tvImg": IMG_PUBLIC + "logos/langGen/Assamese_1579245819981.jpg",
+        "promoImg": IMG_PUBLIC+"67/0/Assamese_1580459753008_promo.jpg",
+    },
+    {
+        "name": "Nepali",
+        "tvImg": IMG_PUBLIC + "logos/langGen/Nepali_1579245819981.jpg",
+        "promoImg": IMG_PUBLIC+"67/0/Nepali_1580459753008_promo.jpg",
+    },
+    {
+        "name": "French",
+        "tvImg": IMG_PUBLIC + "logos/langGen/French_1579245819981.jpg",
+        "promoImg": IMG_PUBLIC+"67/0/French_1580459753008_promo.jpg",
+    },
+    {
+        "name": "Manipuri",
+        "tvImg": IMG_PUBLIC + "logos/langGen/Manipuri_1579245819981.jpg",
+        "promoImg": IMG_PUBLIC+"67/0/Manipuri_1580459753008_promo.jpg",
+    },
+    {
+        "name": "Koshali",
+        "tvImg": IMG_PUBLIC + "logos/langGen/Koshali_1579245819981.jpg",
+        "promoImg": IMG_PUBLIC+"67/0/Koshali_1580459753008_promo.jpg",
     }
 ]
 LANG_MAP = {6: "English", 1: "Hindi", 2: "Marathi", 3: "Punjabi", 4: "Urdu", 5: "Bengali", 7: "Malayalam", 8: "Tamil",
-            9: "Gujarati", 10: "Odia", 11: "Telugu", 12: "Bhojpuri", 13: "Kannada", 14: "Assamese", 15: "Nepali", 16: "French", 18:"Extra"}
+            9: "Gujarati", 10: "Odia", 11: "Telugu", 12: "Bhojpuri", 13: "Kannada", 14: "Assamese", 15: "Nepali", 16: "French", 18:"Manipuri", 19:"Koshali"}
```

Comment:
How do you know 18 is manipuri and 19 is koshali????
as per https://jiotvapi.cdn.jio.com/apis/v1.3/dictionary/dictionary?langId=6

there is only
    "languageIdMapping": {
        "1": "Hindi",
        "2": "Marathi",
        "3": "Punjabi",
        "4": "Urdu",
        "5": "Bengali",
        "6": "English",
        "7": "Malayalam",
        "8": "Tamil",
        "9": "Gujarati",
        "10": "Odia",
        "11": "Telugu",
        "12": "Bhojpuri",
        "13": "Kannada",
        "14": "Assamese",
        "15": "Nepali",
        "16": "French"
    },